forge-gui/res/cardsfolder/upcoming/exterminate.txt

```diff
@@ -0,0 +1,8 @@
+Name:Exterminate!
+ManaCost:2 B
+Types:Sorcery
+K:Replicate:tapXType<1/Dalek>
+A:SP$ Destroy | ValidTgts$ Creature | SubAbility$ DBDrain | SpellDescription$ Destroy target creature. Its controller loses 3 life.
+SVar:DBDrain:DB$ LoseLife | Defined$ TargetedController | LifeAmount$ 3
```

Comment:
OK, I know this is pedantic, but.

Say someone targets a creature with this. The desc goes to the stack. But then someone uses something at instant speed that results in the targeted creature changing controllers. The stack description for line 6 is now wrong.

So I think line 6 should be StackDescription$ SpellDescription and pull the second sentence from line 5 for the SpellDescription

Hopefully this makes sense 😅